### DISCUSSION TITLE
Comparison with cookiecutter

### DISCUSSION THREAD

**Discussion by TejasAvinashShetty** [fortran-lang/fpm#641] [General]
How does fpm compare with cookie-cutter ?
- https://github.com/mdklatt/cookiecutter-fortran-app
- https://github.com/aradi/cookiecutter-fortran-cmake

For example cookie cutter has prompts as to enter the name , email, etc.

**Comment by awvwgk** [comment]
After deploying a cookiecutter template you only use the resulting templete, *i.e.* in case of Bálint's CMake template you will continue building your project with CMake. While fpm is a build system rather than a sole templating engine (we also want to support custom project templates at some point), which can compile, run and install the project as well.
